{{ cookiecutter.format }}/briefcase.toml

```diff
@@ -7,7 +7,8 @@ support_path = "src"
     "3.8": "support_revision = 10",
     "3.9": "support_revision = 13",
     "3.10": "support_revision = 11",
-    "3.11": "support_revision = 3",
+    "3.11": "support_revision = 5",
+    "3.12": "support_revision = 1",
```

Comment:
If this is the micro version number, shouldn't it be 3.12.0?

And did you intend to bump the other micro versions as well?

Reply:
🤦  Yes, it should be 3.12.0 

As for the other releases - those are the most recent releases that actually provide binaries. Once a Python release moves into security update mode they no longer provide binaries (e.g., [this page](https://www.python.org/downloads/release/python-3818/) for 3.8.18.)